src/radha/Assigment12/Assignment12.java

```diff
@@ -0,0 +1,59 @@
+package radha.Assigment12;
+
+public class Assignment12 {
+	
+	double sum(double num1, double num2, double num3) {
+		double sum = num1+num2+num3;
+		return sum;
+	}
+	
+	double average(double num1, double num2, double num3) {
+		double sum2 = sum(num1,num2,num3);
+		double answer = sum2/3;
+		System.out.println("Average is: "+answer);
+		return answer;
+	}
+	
+	boolean isAnswerAboveExpectation(double answer) {
+		if(answer>50)
+			return true;
+		else
+			return false;
+	}
+	
+	static boolean isEligible(double answer) {
+		if((answer>50) || (answer%2==0)) {
```

Comment:
we need to use divisible function not modular.
if value is more than 50 or divisible by 2. Eligibility will be printed in main method.